bundles/org.openhab.binding.easee/README.md

```diff
@@ -0,0 +1,180 @@
+# Easee Binding
+
+The Easee binding can be used to retrieve data from the Easee Cloud API and also to control your wallbox via the Cloud API.
+This allows you to dynamically adjust the charge current for your car depending on production of your solar plant.
+
+## Supported Things
+
+This binding provides three thing types: 
+
+| Thing/Bridge        | Thing Type          | Description                                                                                   |
+|---------------------|---------------------|-----------------------------------------------------------------------------------------------|
+| bridge              | site                | cloud connection to a site within an Easee account                                            |
+| thing               | charger             | the physical charger which is connected to a circuit within the given site                    |
+| thing               | mastercharger       | like the "normal" charger but with additional capability to control the circuit               |
+
+
+Basically any Easee wallbox that supports the Cloud API should automatically be supported by this binding.
+
+## Discovery
+
+Discovery is implemented and will discover all circuits and chargers assigned to a given site.
+
+## Bridge Configuration
+
+The following configuration parameters are available for the binding/bridge:
+
+- **username** (required)
+The username to login at Easee Cloud service.
+This should be an e-mail address or phone number.
+
+- **passord** (required)  
+Your password to login at Easee Cloud service.
+
+- **siteId** (required)  
+The ID of the site containing the wallbox(es) and circuit(s) that should be integrated into openHAB.
+The ID of your site can be found out via the sites overview (https://easee.cloud/sites).
+You just need to click on one of the sites listed there, the id will be part of the URL which is then opened.
+It will be a number with typically 6 digits.
+
+- **dataPollingInterval** (optional)  
+interval (seconds) in which live data values are retrieved from the Easee Cloud API. (default = 120)
```

Comment:
I would suggest to format this as a table for better overview/readability, like many other bindings do. See for example https://www.openhab.org/addons/bindings/hdpowerview/#thing-configuration.